tests/integration/test_records.py

```diff
@@ -306,6 +307,26 @@ def test_merged_records_stay_merged(api_client, merged_records):
     assert api_client.get('/literature/222').status_code == 301
 
 
+def test_merge_record_with_non_existing_pid(api_client, merged_records):
+    def get_pid_entry(recid):
+        return PersistentIdentifier.query.filter_by(pid_value=str(recid)).one_or_none()
+
+    merged_record = get_db_record('lit', 111)
+
+    assert get_recid_from_ref(merged_record['deleted_records'][0]) == 222
+    # remove it so it doesn't exists anymore
```

Comment:
s/exists/exist/

Reply:
doh!

Reply:
I usually do the typos by swapping letters, like 'eixsts' or 'rwapper' xd